src/Control/Monad/Aff.js

```diff
@@ -320,3 +320,11 @@ exports._tailRecM = function (isLeft, f, a) {
     }(a);
   };
 };
+
+
+var avar = require("../Control.Monad.Aff.AVar/foreign.js");
```

Comment:
Hmm, it doesn't exactly work. It works in node if the planets align correctly, as the `require` is resolved by node instead... but the bundle isn't browser-safe at that point, so yeah, this probably isn't going to work.